src/library_gl.js

```diff
@@ -9,7 +9,14 @@
  */
 
 var LibraryGL = {
+#if USE_PTHREADS
+  // If GLctxIsProxied is true, then
+  //   a) the running thread is not the main browser thread, and
+  //   b) the currently active GL context is running on the main browser thread, so all GL calls need to be proxied to it.
+  $GL__postset: 'var GLctx; var GLctxIsProxied = false; GL.init()',
```

Comment:
why do we have `GLctxIsProxied` instead of `GL.ctxIsProxied` (i.e. as a property on GL)?

Reply:
That is good point - at some point I had part of the proxying extend to `src/pthread-main.js`, which is currently outside minification, so it did not see the `GL` variable to access, but looking at it now, I no longer have that, so this no longer needs to be global. Given that we have `GL.currentContext`, I also renamed this once more to `GL.currentContextIsProxied` to make it look similar.